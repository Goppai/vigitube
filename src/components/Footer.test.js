import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { Footer } from './'

describe('<Footer/>', () => {
  const wrapper = shallow(<Footer>
    <span className='footer-child'>text</span>
  </Footer>)

  it('renders', () => {
    expect(toJson(wrapper)).toMatchSnapshot()
  })

  it('renders children', () => {
    expect(wrapper.find('.footer-child').length).toBe(1)
  })
})
