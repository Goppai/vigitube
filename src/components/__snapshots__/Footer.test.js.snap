// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> renders 1`] = `
<Col
  className="center-text footer"
  xs={24}
>
  <Adapter>
    <span
      className="footer-child"
    >
      text
    </span>
  </Adapter>
</Col>
`;
